python/llm/example/transformers/transformers_int4/whisper/readme.md

```diff
@@ -57,4 +58,61 @@ numactl -C 0-47 -m 0 python ./recognize.py
 Inference time: xxxx s
 -------------------- Output --------------------
 [" Mr. Quilter is the Apostle of the Middle classes and we're glad to welcome his Gospel."]
-```
\ No newline at end of file
+```
+
+
+## Example 2: Recognize Long Segment using `generate()` API
+In the example [long-segment-recognize.py](./long-segment-recognize.py), we show a basic use case for a Whisper model to conduct transcription using `pipeline()` API, with BigDL-LLM INT4 optimizations.
```

Comment:
we show a basic use case for a Whisper model to conduct transcription using `pipeline()` API for long audio input, ...

Maybe emphasize long audio input here.